src/Parsers/ExpressionElementParsers.cpp

```diff
@@ -1451,6 +1451,7 @@ const char * ParserAlias::restricted_keywords[] =
     "ASOF",
     "BETWEEN",
     "CROSS",
+    "PASTE",
```

Comment:
Btw, I recently though about it and actually it can be broken change and we need to mention it properly in a changelog. Because `paste` is parsed as a keyword instead of alias name. wee need to consider which queries may stop work if someone uses `paste` as subquery name: `(SELECT ... ) paste ...`, `(SELECT ... ) AS paste ...` and so on, and explain it in a changelog. Maybe even something already changed when we introduced PASTE join, but for me it became clear only after we found this thing with `restricted_keywords`